src/soma.cpp

```diff
@@ -42,7 +42,19 @@ float Soma::surface() const
 
 float Soma::maxDistance() const
```

Comment:
is there a place where duplicate functionality can be called put, so we don't have to maintain two versions?  kind of like how centerOfGravity works

Reply:
Sure. I can follow up.